js_tests/tests.html

```diff
@@ -77,12 +77,17 @@
     <script src='../django/contrib/admin/static/admin/js/prepopulate.js' data-cover></script>
     <script src='../django/contrib/admin/static/admin/js/urlify.js' data-cover></script>
 
-    <div id="id_point_map">
+    <div id="id_point_map" style="display:none;">
         <textarea id="id_point" name="point"
                   class="vSerializedField required" style="display:none;"
-                  rows="10" cols="150">POINT (7.8177 47.397)</textarea>
+                  rows="10" cols="150">{&quot;type&quot;: &quot;Point&quot;, &quot;coordinates&quot;: [7.8177, 47.397]}</textarea>
```

Comment:
adjust for shorter line length? I guess I would use a 4 space hanging indent for lines after `<textarea` to avoid some non-multiple of 4 indent but it's up to you.